bakery-js/src/epub/toc.tsx

```diff
@@ -79,6 +81,16 @@ export class TocFile extends BaseTocFile<
       ? (collectionXml.findOne('//col:collection').attr('authors') as string)
       : 'OpenStax Authors'
 
+    // Check for cover JPEG file
+    const checkCoverFilePath = join(
+      dirname(this.readPath),
+      '..',
+      DIRNAMES.IO_FETCHED,
+      'cover',
+      slug + '-cover.jpg'
```

Comment:
Maybe this is an intentional limitation, but I still wanted to point out that cover images could be jpg or png.

Granted, it looks like there a quite a few things that assume the cover will be a jpg.

Reply:
Yes this is intentional. See https://github.com/openstax/ce/issues/2074

There has to go additional work into cover images in future. Like more explicit schema changes. For now it is limited to JPEG and not check if it's a valid JPEG (not too big, not too small etc).